cisco_aci/tests/fixtures/metadata.py

```diff
@@ -0,0 +1,3385 @@
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/23',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/24',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/24',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/25',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/25',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/26',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/26',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/27',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/27',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/28',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/28',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/29',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/29',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/30',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
+        ],
+        'index': 'eth1/30',
+        'mac_address': 'not-applicable',
+        'name': '',
+        'status': 1,
+    },
+    {
+        'device_id': 'default:10.0.200.5',
+        'id_tags': [
+            'port:eth1/31',
+            'medium:broadcast',
+            'snmpTrapSt:enable',
+            'node_id:201',
+            'fabric_pod_id:1',
```

Comment:
The id_tags seems weird to me, what tags uniquely identify the interface?

Reply:
i believe the port: `eth1/31` would be the unique ID for each interface on cisco aci !
i just propagated these for the tags that we put on the interface metrics thinking that they may be applicable / are ACI components-relevant